bindings/pyroot/pythonizations/test/CMakeLists.txt

```
@@ -187,11 +187,10 @@ if (dataframe)
     # std::string_view in CPyCppyy
     ROOT_ADD_PYUNITTEST(pyroot_string_view string_view.py)
     if(NOT MSVC OR win_broken_tests)
-        if(NOT (DEFINED ENV{ROOTTEST_IGNORE_NUMBA_PY2} AND PYTHON_VERSION_MAJOR_Development_Main EQUAL 2) AND NOT
-            (DEFINED ENV{ROOTTEST_IGNORE_NUMBA_PY3} AND PYTHON_VERSION_MAJOR_Development_Main EQUAL 3))
+        if(NOT (DEFINED ENV{ROOTTEST_IGNORE_NUMBA_PY3} AND PYTHON_VERSION_MAJOR EQUAL 3))
```

```suggestion
        if(NOT (DEFINED ENV{ROOTTEST_IGNORE_NUMBA}))
```
Do we really need to check for Python 3 explicitly here?